src/codegen/utilities/helpers.py

```diff
@@ -30,3 +59,35 @@ def get_enums_to_import(enums_in_attributes, enums_in_functions):
 def strip_class_name(name: str, class_name: str, replace_with=""):
     """Strips class name from name."""
     return re.sub(class_name, replace_with, name)
+
+
+def snake_to_pascal(snake_string):
```

Comment:
FYI: This is kind of surprising, considering that the metadata has the original PascalCase names: https://github.com/ni/nidaqmx-python/blob/master/src/codegen/metadata/functions.py#L12250

I see you copied this from https://github.com/ni/grpc-device/blob/main/source/codegen/common_helpers.py

@astarche @reckenro I suppose the PascalCase names generated by `protoc` don't always match the drivers' original PascalCase names?

Reply:
I'm marking this as resolved.